tests/e2e/pubsub/pubsub_test.go

```diff
@@ -0,0 +1,189 @@
+// +build e2e
+
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package pubsubapp_e2e
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"reflect"
+	"testing"
+
+	
+	"github.com/dapr/dapr/tests/e2e/utils"
+	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
+	"github.com/dapr/dapr/tests/runner"
+	"github.com/stretchr/testify/require"
+)
+
+const numHealthChecks = 3      // Number of get calls before starting tests.
+const testManyEntriesCount = 5 // Anything between 1 and the number above (inclusive).
```

Comment:
The comment seems to be outdated. There is no dependency between this and the const above.